### PR TITLE
Create ggwilliams.md

### PR DESCRIPTION
**Pull request by ggwilliams**
[Rawgit Link](https://rawgit.com/ggwilliams/ggwilliams.github.io/ggwilliams/#!pages/profiles/ggwilliams.md)

### CODE REVIEW

File: pages/profiles/ggwilliams.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+My name is *Gabby* I just finished the General Assembly WDI program in **Boston** where I learned the following languages:
+1. Javascript
+2. Ruby on Rails
+3. Ember
+4. and more...
+
+I hope to learn a lot more through [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io "Open Learning Exchange")
+
+I enjoy:
+*FrontEnd
```
Comment (sarah-lu102) [inline pages/profiles/ggwilliams.md]: the bullet formatting only shows up when you put a space between "*" and the text.

**Comment by navneetkarnam** [comment]
@ggwilliams are you still working on this PR?

File: pages/profiles/ggwilliams.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+I enjoy:
+* FrontEnd 
+* Javascript Algorithms
+* Code Challenges
+
+When I have ~~freetime~~, I like to:
+* Practice more coding problems
+* Watch tv
+* Play with my dogs
+* Hang out with familt
```
Comment (kylemathias) [inline pages/profiles/ggwilliams.md]: Spelling.

File: pages/profiles/ggwilliams.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+I hope to learn a lot more through [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io "Open Learning Exchange")
+This is my project portfolio site [portfolio](https://ggwilliams.github.io/portfolio//)
+
+I enjoy:
+* FrontEnd 
+* Javascript Algorithms
+* Code Challenges
+
+When I have ~~freetime~~, I like to:
```
Comment (kylemathias) [inline pages/profiles/ggwilliams.md]: When you read it without the cross out it reads "When i have, i like to:". Could change it to "When i have ~~free~~time, i like to:"

**Comment by alexkarasik** [comment]
Please continue with the rest of the steps if you are still doing the firststeps @ggwilliams

**Pull request closed without merging**